### PR TITLE
Fix allow on submit for status

### PR DESCRIPTION
**Pull request by Alchez**

### CODE REVIEW

File: erpnext/patches/v11_0/update_delivery_trip_status.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+from __future__ import unicode_literals
+import frappe
+
+def execute():
+	frappe.reload_doc('stock', 'doctype', 'delivery_trip')
+
+	for trip in frappe.get_all("Delivery Trip"):
+		trip_doc = frappe.get_doc("Delivery Trip", trip.name)
+
+		status_map = ["Draft", "Scheduled", "Cancelled"]
```
Comment (vjFaLk) [inline erpnext/patches/v11_0/update_delivery_trip_status.py]: I let out an audible "wat" cause it took me a while to understand this, I know this **works** but this needs to be a dict.

**Review by vjFaLk — CHANGES_REQUESTED** [review]

File: erpnext/patches/v11_0/update_delivery_trip_status.py (hunk 1 above)
Comment (Alchez) [reply erpnext/patches/v11_0/update_delivery_trip_status.py]: But why though? It's just an index map anyway. :)

Comment (vjFaLk) [reply erpnext/patches/v11_0/update_delivery_trip_status.py]: Dunno, docstatus is a confusing thing as it is, code around it should at least be more clear

Comment (Alchez) [reply erpnext/patches/v11_0/update_delivery_trip_status.py]: Hm, fair enough. I'll make a change.

**Comment by vjFaLk** [comment]
@Alchez Is this going to Upstream? Cause it needs to

**Comment by Alchez** [comment]
Already added it to the existing PR there.

**Pull request merged**